src/Tribe/Aggregator/Record/Eventbrite.php

```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
 	 * @return array
 	 */
 	public static function filter_setup_do_not_override_post_status( $post_status, $event, $record ) {
-
-		if ( ! isset( $event['eventbrite']->status ) || ! isset( $record->meta['post_status'] ) ) {
-			return $post_status;
-		}
-
-		// We're only interested in the "(do not override)" post status.
-		if ( 'do_not_override' !== $record->meta['post_status'] ) {
-			return $post_status;
-		}
-
-		// Currently, 'live' is the only Eventbrite.com status that needs mapping to a WP post status.
-		if ( 'live' === $event['eventbrite']->status ) {
-			return 'publish';
+		// override status if set within import.
+		$status = isset( $record->meta['post_status'] ) ? $record->meta['post_status'] : $post_status;
+		if ( 'do_not_override' === $status ) {
+			if ( isset( $event['eventbrite']->status ) ) {
+				// draft if set, otherwise publish.
+				if ( 'draft' === $event['eventbrite']->status ) {
+					$status = 'draft';
+				} else {
+					$status = 'publish';
+				}
+			} else {
+				// if not set, assume publish.
+				$status = 'publish';
+			}
+			// If not draft, looked if listed. If not, set to private.
+			if ( 'draft' ! == $status && isset( $event['eventbrite']->listed ) && !tribe_is_truthy( $event['eventbrite']->listed ) ) {
```

Comment:
* :no_entry_sign: PHP syntax error: syntax error, unexpected '!'
* :no_entry_sign: Expected 1 space after "!"; 0 found
* :no_entry_sign: ! must be followed by a single space

<sub>hash: be9ec11c193e4b7bdee79318f8664aef</sub>